src/python/txtai/pipeline/text/translation.py

```diff
@@ -52,7 +54,7 @@ def __init__(self, path="facebook/m2m100_418M", quantize=False, gpu=True, batch=
         self.models = {}
         self.ids = self.modelids()
 
-    def __call__(self, texts, target="en", source=None):
+    def __call__(self, texts, target="en", source=None, debug=False):
```

Comment:
I think this will be useful for more than just debugging. How about we name the new parameter `showmodels`.

Reply:
Done!